src/main/resources/config/application-dev.yml

```diff
@@ -74,10 +74,10 @@ jhipster:
 application:
     adminApiRestrictionEnabled: true
     git:
-        uri:
+        uri: /Users/serhii.senko/work/XM/xm-ms-config-repository
```

Comment:
it is better to use ${user.home}/work/XM/xm-ms-config-repository or did not commit at all